wg-device-management/charter.md

```diff
@@ -0,0 +1,3 @@
+# WG Device Management
+
+In progress.
```

Comment:
:)

Reply:
Yes, I wasn't clear from https://github.com/kubernetes/community/blob/master/sig-wg-lifecycle.md whether the charter can follow merge of this or we should put it together now. That is, whether writing the charter should be the first thing we do as a WG, or we need it first.

Reply:
charter should be the first step with scope and exit criteria